src/validation.cpp

```diff
@@ -5106,6 +5110,10 @@ bool LoadMempool(CTxMemPool& pool)
         for (const auto& i : mapDeltas) {
             if (FeeDeltaRange(i.second)) {
                 pool.PrioritiseTransaction(i.first, i.second);
+            } else {
+                if (i.second != 0) {
```

Comment:
This condition is always true, isn't it? 0 is a valid fee delta, hence the else-branch wouldn't be taken in the case that i.second is zero.

Reply:
You're absolutely right: the conditional is redundant. Thanks for noticing! Redundancy removed :)

Please re-review! :)